lisa/environment.py

```diff
@@ -345,6 +350,10 @@ def get_information(self) -> Dict[str, str]:
 
         return final_information
 
+    def mark_dirty(self) -> None:
+        self.log.debug("mark environment to dirty")
```

Comment:
Can we add environment name to improve logging information?

Reply:
Because this log is written to `self.log`. It has the "env[env_name]" leading the line already.